packages/snap-preact-components/src/components/Organisms/BundledRecommendations/BundledRecommendations.tsx

```diff
@@ -0,0 +1,515 @@
+														quantity={showQuantityPicker ? result.quantity : undefined}
+														showCheckboxes={showCheckboxes}
+													>
+														<RecommendationResultTracker controller={controller} result={result}>
+															{resultComponent ? (
+																cloneWithProps(resultComponent, { result: result, seed: true, selected, onProductSelect })
+															) : (
+																<Result {...subProps.result} controller={controller} result={result} />
+															)}
+														</RecommendationResultTracker>
+													</BundleSelector>
+												</div>
+											);
+										} else {
+											return (
+												<BundleSelector
+													icon={seedIconOnly ? false : seperatorIcon}
+													onCheck={() => onProductSelect(result)}
+													checked={selected}
+													onInputChange={(e) => onInputChange(result, e.target.value)}
+													quantity={showQuantityPicker ? result.quantity : undefined}
+													showCheckboxes={showCheckboxes}
+												>
+													<RecommendationResultTracker controller={controller} result={result}>
+														{resultComponent ? (
+															cloneWithProps(resultComponent, { result: result, seed: false, selected, onProductSelect })
+														) : (
+															<Result {...subProps.result} controller={controller} result={result} />
+														)}
+													</RecommendationResultTracker>
+												</BundleSelector>
+											);
+										}
+									})}
+								</Carousel>
+							</div>
+							{!isMobile && (
+								<BundledCTA
+									isMobile={isMobile}
+									ctaSlot={ctaSlot}
+									selectedItems={selectedItems}
+									bundlePrice={bundlePrice}
+									bundleStrikePrice={bundleStrikePrice}
+									addToCartFunc={(e: any) => addToCart(e)}
+									addToCartText={addToCartText}
+								/>
+							)}
+						</div>
+						{isMobile && (
+							<BundledCTA
+								isMobile={isMobile}
+								ctaSlot={ctaSlot}
+								selectedItems={selectedItems}
+								bundlePrice={bundlePrice}
+								bundleStrikePrice={bundleStrikePrice}
+								addToCartFunc={(e: any) => addToCart(e)}
+								addToCartText={addToCartText}
+							/>
+						)}
+					</div>
+				</RecommendationProfileTracker>
+			</div>
+		</CacheProvider>
+	) : (
+		<Fragment></Fragment>
+	);
+});
+
+export interface BundledRecommendationsProps extends ComponentProps {
+	results?: Product[];
+	controller: RecommendationController;
+	onAddToCart: (items: selectedItem[]) => void;
+	addToCartText?: string;
+	title?: JSX.Element | string;
+	breakpoints?: BreakpointsProps;
+	prevButton?: JSX.Element | string;
+	nextButton?: JSX.Element | string;
+	hideButtons?: boolean;
+	loop?: boolean;
+	pagination?: boolean;
+	resultComponent?: JSX.Element;
+	ctaSlot?: JSX.Element;
+	preselectedCount?: number;
+	showQuantityPicker?: boolean;
+	showCheckboxes?: boolean;
+	seedInCarousel?: boolean;
+	seedText?: string;
+	seedIconOnly?: boolean;
```

Comment:
I wonder if something more like `hideCarouselSeparatorIcons` or similar would be more clear? Or maybe just rename to `seedSeparatorIconOnly`?